.changeset/yellow-houses-talk.md

```diff
@@ -0,0 +1,77 @@
+---
+"@comet/admin-color-picker": major
+"@comet/admin-react-select": major
+"@comet/admin-date-time": major
+"@comet/blocks-admin": major
+"@comet/admin-theme": major
+"@comet/admin-rte": major
+"@comet/cms-admin": major
+"@comet/admin": major
+---
+
+Change the method of overriding the styling of admin components
+
+-   Remove dependency on the legacy `@mui/styles` package in favor of `@mui/material/styles`.
+-   Add the ability to style components using [MUI's `sx` prop](https://mui.com/system/getting-started/the-sx-prop/).
+-   Add the ability to style individual elements (slots) of a component using the `slotProps` and `sx` props.
+-   The `$` syntax in the theme's `styleOverrides` is no longer supported, see: https://v5-0-6.mui.com/guides/migration-v4/#migrate-themes-styleoverrides-to-emotion
```

Comment:
Shouldn't we use the most up-to-date guide here? https://mui.com/material-ui/migration/v5-style-changes/#migrate-theme-styleoverrides-to-emotion